src/main/java/com/mitchej123/hodgepodge/net/HandlerConfigSync.java

```diff
@@ -0,0 +1,17 @@
+package com.mitchej123.hodgepodge.net;
+
+import com.mitchej123.hodgepodge.Common;
+
+import cpw.mods.fml.common.network.simpleimpl.IMessage;
+import cpw.mods.fml.common.network.simpleimpl.IMessageHandler;
+import cpw.mods.fml.common.network.simpleimpl.MessageContext;
+
+public class HandlerConfigSync implements IMessageHandler<MessageConfigSync, IMessage> {
```

Comment:
Optional: many mods implement `IMessage` and `IMessageHandler` with one class

Reply:
Can you show me an example?

Reply:
Found one and implemented